spectrum.js

```diff
@@ -857,12 +857,19 @@
         }
 
         function updateOriginalInput(fireCallback) {
-            var color = get(),
+            var format = currentPreferredFormat;
```

Comment:
I'd prefer if we had a new getCurrentPreferredFormat() function so we can share this logic with updateUI where the preview text is set